build/scripts/dist.xml

```diff
@@ -305,6 +312,17 @@
                 <include name="wrapper"/>
             </fileset>
         </chmod>
+        
+        <!-- Re-sign jar files that get re-built when this script gets invoked -->
```

Comment:
I don;t think we can rely on the resign here; when needed at al, better is to call the resign-tasks (which verifies for the availability of the keystore file. just calling resign is not a good idea